src/components/scene/ar-hit-test.js

```diff
@@ -37,6 +61,36 @@ HitTest.prototype.sessionStart = function sessionStart (options) {
   }
 };
 
+HitTest.prototype.anchorFromLastHitTestResult = function (options) {
+  var hitTest = this.lastHitTest;
+
+  if (!hitTest) {
+    return;
+  }
+
+  var object3D = options.object;
+
+  Array.from(anchorToObject3D.entries())
+  .forEach(function (entry) {
+    var entryObject = entry[1].object;
+    var anchor = entry[0];
+    if (entryObject === object3D) {
+      anchorToObject3D.delete(anchor);
+      anchor.delete();
+    }
+  });
+
+  if (hitTest.createAnchor) {
+    hitTest.createAnchor()
+    .then(function (anchor) {
+      anchorToObject3D.set(anchor, options);
```

Comment:
Is better to use a map vs. a regular object here? object3D has an id that could be used as object key. I don't have much experience working with Maps.

Reply:
We're already using Maps and they make more semantic sense in this case since it is how they are supposed to be used.